assets/js/src/newsletter_editor/initializer.jsx

```diff
@@ -29,7 +29,7 @@ const initializeEditor = (config) => {
     !parseInt(window.mailpoet_woocommerce_settings.use_mailpoet_editor, 10)
     && getUrlParam('id') === window.mailpoet_woocommerce_settings.transactional_email_id
   ) {
-    window.location.href = 'admin.php?page=mailpoet-settings#woocommerce';
+    window.location.href = 'admin.php?page=mailpoet-settings#woocommerce/enable-notice';
```

Comment:
The URL sounds like "enable WooCommerce" notice. I'd suggest something like `#woocommerce/enable-customizer-notice`.